omod/src/main/webapp/resources/htmlforms/Mobility-Screening-form.xml

```diff
@@ -0,0 +1,106 @@
+<htmlform formName="Mobility Screening form"
+          formEncounterType="b57e1835-4ee2-47fa-9569-c700c39c169a"
+          formDescription="Mobility Screening form"
+          formUuid="67a71486-2a54-468f-ac3e-BB91a9a79545"
+          formVersion="1.0"
+          formAddMetadata="yes"
+          formUILocation="patientDashboard.visitActions"
+          formOrder="1"
+          formIcon="icon-stethoscope"
+          formShowIf="(visit.active || !visit.active) &amp;&amp; patient.person.dead==false"
+          formDisplayStyle="Standard"
+          formLabel="Mobility Screening form">
+
+    <script type="text/javascript">
+
+        var facilityName = '<lookup expression="fn.globalProperty('aijar.healthCenterName')" />';
```

Comment:
did you mean to say `ugandaemr.healthCenterName` instead of `aijar.healthCenterName`